pkg/util/mount/mount.go

```diff
@@ -97,40 +104,22 @@ type HostUtils interface {
 	GetMode(pathname string) (os.FileMode, error)
 }
 
-type Subpath struct {
```

Comment:
This was mistakenly leftover from when `subpath` was moved into `pkg/volume/util`. It is unused here.